unit-of-work/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>junit-vintage-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.0.0</version>
```

Comment:
Use the one from parent pom.xml

Reply:
If you would like me to use the version from the parent pom, then the version within the parent pom would have to be updated, as that version is causing build issues within this module.

Reply:
The other issues have been resolved and I will not push any changes just yet, as I'd like to know if you want to open a new issue regarding updating the classes that rely on the older version of **`junit-jupiter-engine`**.